contexts/credentials/v2

```diff
@@ -266,6 +266,10 @@
           "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
         },
         "nonce": "https://w3id.org/security#nonce",
+        "previousProof": {
+          "@id": "https://w3id.org/security#previousProof",
```

Comment:
@dlongley can this be used with Verifiable Presentations as well?

Reply:
Yes.